### PR TITLE
Chrome removed support for X-XSS-Protection header

### PR DESCRIPTION
**Pull request by antonjuulnaber**
https://www.chromium.org/developers/design-documents/xss-auditor
https://chromestatus.com/feature/5021976655560704

Related: #4525

### CODE REVIEW

**Comment by antonjuulnaber** [comment]
Allright, will do.

File: http/headers/x-xss-protection.json [hunk 1]
```diff
@@ -25,7 +27,8 @@
               "version_added": "8"
             },
             "opera": {
-              "version_added": true
+              "version_added": true,
+              "version_removed": "56"
```
Comment (vinyldarkscratch) [inline http/headers/x-xss-protection.json]: Whoops!  Looks like you might have swapped the digits.  😛 

```suggestion
              "version_removed": "65"
```

**Comment by antonjuulnaber** [comment]
Absolutely. I didn't know that Opera was so aligned with Chrome. Will remember in the future 😊

File: http/headers/x-xss-protection.json (hunk 1 above)
Comment (antonjuulnaber) [reply http/headers/x-xss-protection.json]: Arh, dangit 😄

**Pull request merged**